tools/cli/commands.js

```diff
@@ -915,11 +931,26 @@ main.registerCommand({
     // meteor create app-name
     if (options.args.length === 1) {
       const appPathAsEntered = options.args[0];
-      const skeletonExplicitOption =
-        AVAILABLE_SKELETONS.find(skeleton => !!options[skeleton]);
-
-      const skeleton = skeletonExplicitOption || DEFAULT_SKELETON;
-
+      
+      // Check for TypeScript + Framework combinations first
+      let skeleton;
+      if (options.typescript && options.vue) {
+        skeleton = 'vue-typescript';
+      } else if (options.typescript && options.apollo) {
+        skeleton = 'apollo-typescript';
+      } else if (options.typescript && options.blaze) {
+        skeleton = 'blaze-typescript';
+      } else if (options.typescript && options.solid) {
+        skeleton = 'solid-typescript';
+      } else if (options.typescript && options.svelte) {
+        skeleton = 'svelte-typescript';
+      } else {
+        // Check for explicit skeleton option or use default
+        const skeletonExplicitOption =
+          AVAILABLE_SKELETONS.find(skeleton => !!options[skeleton]);
+        skeleton = skeletonExplicitOption || DEFAULT_SKELETON;
+      }
```

Comment:
I would prefer something more in a functional style:

```suggestion
const skeleton = (() => 
       if (options.typescript) {
          return AVAILABLE_TYPESCRIPT_SKELETONS.find(skeleton => !!options[skeleton]);
       } else { AVAILABLE_SKELETONS.find(skeleton => !!options[skeleton]); }
      )()
```